net-p2p/bitcoin-core/bitcoin-core-25.1-r1.ebuild

```diff
@@ -254,6 +305,14 @@ pkg_postinst() {
 			- Running bitcoind directly: add that user to the 'tor' user group.
 			EOF
 	fi
+
+	if use bitcoin-cli && use daemon ; then
+		efmt -su elog <<-EOF
+			To use bitcoin-cli with the /etc/init.d/bitcoind service:
+			 - Add your user(s) to the 'bitcoin' group.
+			 - Symlink ~/.bitcoin to /var/lib/bitcoind.
```

Comment:
```suggestion
			 - Symlink ~/.bitcoin to ${EROOT}/var/lib/bitcoind.
```

Reply:
Even if you have not strong feelings about `${EROOT}/etc/init.d/bitcoind`, you must have strong feelings about this one. Symlinking the default data dir in one's home directory to `${EROOT}/var/lib/bitcoind` makes no sense, as one won't be calling `bitcoin-cli` on a system rooted at `/` to access a `bitcoind` instance running on a system rooted at `${EROOT}`. If anything, the message would need to say `Symlink ${EROOT}/${HOME}/.bitcoin to /var/lib/bitcoind`, but that's likely to be wrong too, as the user's home directory might not even exist in `${EROOT}`.

Reply:
Yes, this one I did actually feel something about. But I couldn't really figure out a way to express the `${HOME}` part correctly.

> Symlinking the default data dir in one's home directory to ${EROOT}/var/lib/bitcoind makes no sense, as one won't be calling bitcoin-cli on a system rooted at / to access a bitcoind instance running on a system rooted at ${EROOT}

Sure. The idea would be you'd use that path in a relative symlink in your homedir when setting things up. But I take the point that in general, using `${EROOT}` in output messages is fraught with edge cases.

Reply:
And if we did try to handle those cases, then we would have to start patching `${EPREFIX}` into the default paths (for conf file, data dir, pid file, log dir, etc.) in `/etc/{init,conf}.d/bitcoind`. It's a whole can of worms where I believe it is impossible to correctly anticipate what the user will want to do in an esoteric setup. Best to let them make the adjustments themselves.

Reply:
Yes, quite right. I wish OpenRC properly supported Prefix but that's another matter..

Reply:
There is an obvious alternative: only output this instructional message if `[[ -z "${EROOT}" ]]`. 🤔

Reply:
... oops, I should've thought of that. It avoids noise for `ROOT` users and it stops me from making a suggestion like this without thinking again. But I don't mind if you want to leave it too.